core/src/main/java/io/questdb/cairo/wal/WalWriter.java

```diff
@@ -458,7 +454,7 @@ public void rollUncommittedToNewSegment() {
                 try {
                     final int timestampIndex = metadata.getTimestampIndex();
                     LOG.info().$("rolling uncommitted rows to new segment [wal=")
-                            .$(path).$(Files.SEPARATOR).$(segmentId)
+                            .$(path).$(Files.SEPARATOR).$(newSegmentId)
```

Comment:
newSegmentId is logged next, this is supposed to be the old segment id.

Reply:
I found that log line quite confusing and I initially thought it highlighted a bug where the `segmentId` wasn't being tracked properly. I'll adjust the message slightly to make it ess confusing and move the params back to what they were.